### PR TITLE
implement and teset ThingIFAPI.getTargetState.

### PR DESCRIPTION
**Pull request by naoki-akimoto**
ThingIFAPI.getTargetStateの2種(Alias有無)を実装しスモールテストを追加しました。
レビューをお願いします。

### CODE REVIEW

**Review by yongpingchen — APPROVED** [review]
LGTM

File: ThingIFSDK/ThingIFSDK/ThingIFAPI+GetState.swift [hunk 1]
```diff
@@ -9,39 +9,50 @@
...
+        guard let target = self.target else {
+            completionHandler(nil, ThingIFError.targetNotAvailable)
+            return
+        }
+
+        self.operationQueue.addHttpRequestOperation(
+            .GET,
+            url: "\(baseURL)/thing-if/apps/\(appID)/targets/\(target.typedID.toString())/states",
+            requestHeader:
+            self.defaultHeader + ["Content-Type" : "application/json" ],
```
Comment (dairyo) [inline ThingIFSDK/ThingIFSDK/ThingIFAPI+GetState.swift]: @naoki-akimoto Is `Content-Type` required? I checked the [specification](https://docs.google.com/document/edit?hgd=1&id=1TLZvjKvwSQ_C66XcC7haef5-PtO639y2fjhQIlOBCYE#heading=h.a82wsgp1p2a2) and I did not found `Content-Type`. Could you double check it? 

If `Content-Type` is not required, Please remove this.

**Review by dairyo — CHANGES_REQUESTED** [review]
@naoki-akimoto Could you move 2 `getTargetState` methods here? As as result, We can remove private `_getTargetState` methods.

**Comment by naoki-akimoto** [comment]
@shinsuke-nara fixed. Please confirm.

**Review by dairyo — APPROVED** [review]

**Pull request merged**